### PR TITLE
Add inst select home search

### PR DESCRIPTION
**Pull request by mengesser**
Adding Mees' solution for returning multiple instruments.

### CODE REVIEW

**Comment by mfixstsci** [comment]
@mengesser had to clean up some stuff from your original pull request but now all of the tests are passing, just need to test on one of the servers to verify!

**Comment by mfixstsci** [comment]
@BradleySappington @bhilbert4 this is ready for review!

**Comment by bhilbert4** [comment]
I'm getting an error when I test this using a program that does not contain multiple instruments. (e.g. 1068 is nircam-only).

File: jwql/utils/constants.py [hunk 1]
```diff
@@ -238,6 +238,9 @@
                                       'G235H', 'G395H', 'PRISM']
                           }
 
+# Filename extensions for guider data
+GUIDER_FILENAME_TYPE = ['guider', 'gs-fg', 'gs-track', 'gs-id', 'gs-acq1', 'gs-acq2']
```
Comment (bhilbert4) [inline jwql/utils/constants.py]: 'guider' will be in science files, while the 'gs-' options will be in guide star data. So it seems like you want to take out the 'guider' option given the way you look for matches in forms.py, right?

**Comment by mfixstsci** [comment]
@bhilbert4 should be fixed now.

File: jwql/utils/constants.py (hunk 1 above)
Comment (mfixstsci) [reply jwql/utils/constants.py]: Thanks for information @bhilbert4, fixed this in [f0c68d8](https://github.com/spacetelescope/jwql/pull/1010/commits/f0c68d8efe2f6f431b1363af00073bfb89a2bcd0)

**Comment by mfixstsci** [comment]
Okay @bhilbert4 I have all of the updated changes on dev. I tested your example: `01068` (NRC only) and my example `02739` which contains MIR and NRC data but excludes the guider data.

**Comment by bhilbert4** [comment]
Functionally this looks good! I tested with proposals that use various combinations of instruments, and all the returned values looked good.

**Comment by mfixstsci** [comment]
Thanks @bhilbert4!

**Review by bhilbert4 — APPROVED** [review]
This looks good to me. Nice work! I think this box is going to get a lot of use.

**Pull request merged**